packages/components/src/components/UtilityCard/UtilityCard.tsx

```diff
@@ -0,0 +1,123 @@
+import React from "react";
+import { Box } from "../../primitives/Box";
+import { Badge } from "../Badge";
+import { Icon } from "../Icon";
+import { Text } from "../../primitives/Text";
+
+export enum InteractiveElementType {
+  Card = "card",
+}
+
+export type UtilityCardProps = {
+  /** Sets the card image source */
+  imageSrc: string;
+  /** Sets the title to be rendered as h2 */
+  title: string;
+  /** Sets the type of the clickable element */
+  interactiveElementType?: InteractiveElementType;
+  /** Text describing the kind of category provided */
+  categoryTag: string;
+  /** Text describing the current status of the category as a badge */
+  status?: string;
+  /** Used by StyledComponents to render the component as a specific tag. If href is passed it'll be rendered as "a" */
+  as?: React.ComponentProps<typeof Box.div>["as"];
+  /** Callback function to be used to invoke onClicks */
+  onClick?: React.MouseEventHandler;
+};
+
+const cardBg = {
+  _: "colorBackgroundWeakest",
+};
+
+const cardBorder = {
+  focus: "borderInfo",
+};
+
+const interactiveBg = {
+  active: "colorBackgroundWeak",
+  hover: "colorBackgroundWeak",
+  ...cardBg,
+};
+
+export const UtilityCard: React.FC<UtilityCardProps> = ({
+  as = "div",
+  imageSrc,
+  interactiveElementType,
+  title,
+  categoryTag,
+  status,
+  onClick,
+}) => {
+  const isCardInteractive =
+    InteractiveElementType.Card === interactiveElementType;
+
+  const interactiveElementProps = {
+    onClick,
+    cursor: "pointer",
+    border: 0,
+  };
+
+  return (
+    <Box.div
+      as={isCardInteractive ? "button" : as}
+      backgroundColor={isCardInteractive ? interactiveBg : cardBg}
+      borderColor={isCardInteractive ? cardBorder : null}
+      borderRadius="borderRadius30"
+      display="flex"
+      flexDirection={{ lg: "row" }}
+      padding="space50"
```

Comment:
Adding `outlineColor={{ focus: "colorBorderPrimary" }}` here should fix the focus issue.